components/dashboard/src/admin/UserDetail.tsx

```diff
@@ -19,42 +19,21 @@ import { getGitpodService } from "../service/service";
 import { WorkspaceSearch } from "./WorkspacesSearch";
 import Property from "./Property";
 import { AdminPageHeader } from "./AdminPageHeader";
-import { BillingMode } from "@gitpod/gitpod-protocol/lib/billing-mode";
-import { AttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
 import { CheckboxInputField, CheckboxListField } from "../components/forms/CheckboxInputField";
-import { CostCenterJSON, CostCenter_BillingStrategy } from "@gitpod/gitpod-protocol/lib/usage";
 import { Heading2, Subheading } from "../components/typography/headings";
 
 export default function UserDetail(p: { user: User }) {
     const [activity, setActivity] = useState(false);
     const [user, setUser] = useState(p.user);
-    const [billingMode, setBillingMode] = useState<BillingMode | undefined>(undefined);
-    const [costCenter, setCostCenter] = useState<CostCenterJSON>();
-    const [usageBalance, setUsageBalance] = useState<number>(0);
-    const [usageLimit, setUsageLimit] = useState<number>();
-    const [editSpendingLimit, setEditSpendingLimit] = useState<boolean>(false);
-    const [creditNote, setCreditNote] = useState<{ credits: number; note?: string }>({ credits: 0 });
-    const [editAddCreditNote, setEditAddCreditNote] = useState<boolean>(false);
     const [editFeatureFlags, setEditFeatureFlags] = useState(false);
     const [editRoles, setEditRoles] = useState(false);
     const userRef = useRef(user);
 
     const initialize = () => {
         setUser(user);
-        const attributionId = AttributionId.render(AttributionId.create(user));
-        getGitpodService().server.adminGetBillingMode(attributionId).then(setBillingMode);
```

Comment:
Why no longer show BillingMode? I think that should stay here. :upside_down_face:

Reply:
Ah sorry, ma bad. Ignore.

Reply:
:thought_balloon: A list of orgs a user is member of would be nice, in the future.